src/hocs/withClickOutside.js

```diff
@@ -11,24 +10,21 @@ const descendsFrom = (descendant, ancestor) => {
 	}
 };
 
-const withClickOutside = lifecycle({
-	componentDidMount() {
-		this.innerRef = ReactDOM.findDOMNode(this);
-		this.handler = event => {
-			if (!descendsFrom(event.target, this.innerRef)) {
-				(this.props.onClickOutside || (() => {}))(event);
+const withClickOutside = Comp => ({ onClickOutside = () => {}, ...props }) => {
```

Comment:
Nice!  So, if a component needs to know when a click is outside of it, this hoc can be used.

Reply:
Yep, it's been in for a while, I just rebuiilt it to eliminate recompose and its deprecation warnings.